src/action_request.cc

```diff
@@ -40,7 +40,7 @@
 #include "util/grb_net.h"
 #include "util/tools.h"
 
-ActionRequest::ActionRequest(std::shared_ptr<UpnpXMLBuilder> xmlBuilder, std::shared_ptr<ClientManager> clients, UpnpActionRequest* upnpRequest)
+ActionRequest::ActionRequest(const std::shared_ptr<UpnpXMLBuilder>& xmlBuilder, const std::shared_ptr<ClientManager>& clients, UpnpActionRequest* upnpRequest)
```

Comment:
They were deliberately removed because of memory management forgetting about the objects beeing passed. Check out history of file, please.

Reply:
I looked at this. Looks like an std::move is missing. Once added, clang-tidy does not transform the first parameter. What do you think?